cmd/aws-iam-authenticator/root.go

```diff
@@ -115,7 +118,16 @@ func getConfig() (config.Config, error) {
 	if err := viper.UnmarshalKey("server.mapAccounts", &cfg.AutoMappedAWSAccounts); err != nil {
 		logrus.WithError(err).Fatal("invalid server account mappings")
 	}
-
+	var reservedPrefixConfig []config.ReservedPrefixConfig
+	if err := viper.UnmarshalKey("server.reservedPrefixConfig", &reservedPrefixConfig); err != nil {
+		return cfg, fmt.Errorf("invalid reserved prefix config: %v", err)
+	}
+	if len(reservedPrefixConfig) > 0 {
+		cfg.ReservedPrefixConfig = make(map[string]config.ReservedPrefixConfig)
+		for _, c := range reservedPrefixConfig {
+			cfg.ReservedPrefixConfig[c.BackendMode] = c
```

Comment:
Can you lowercase this key before storage?

Reply:
I have removed the lowercase. It is not needed actually. in config file, 'DynamicFile` is passed in https://github.com/kubernetes-sigs/aws-iam-authenticator/pull/568/files#diff-093cb716b58164ce7c35a6cc848450d2eed7a799a55d3a2e863c0d7cd14a1b36R11